src/modules/aws_rds/entity/parameter_group.ts

```diff
@@ -58,38 +61,81 @@ export enum ParameterGroupFamily {
   SQLSERVER_WEB_15_0 = 'sqlserver-web-15.0',
 }
 
+/**
+ * Table to manage AWS RDS parameter groups
+ *
+ * @example
+ * ```sql
+ * INSERT INTO parameter_group (name, family, description) VALUES ('pg_name', 'postgres14', 'description');
+ * SELECT params ->> 'ParameterValue' as value FROM parameter_group, jsonb_array_elements(parameters) as params
+ * WHERE name = 'pg_name' AND params ->> 'DataType' = 'boolean' AND params ->> 'IsModifiable' = 'true';
+ * DELETE FROM parameter_group WHERE name = 'pg_name';
+ * ```
```

Comment:
is it possible to make these `TheButton` arguments in the typedoc plugin so we can move everything from https://iasql.com/docs/aws_rds/ and then centralize all the documentation here?